### ISSUE TITLE
Use Semantic Versioning

### ISSUE THREAD

**Issue by Datamance** [lookit/lookit-api#558]
**TL;DR**: Lookit-API (and FP for that matter) should be using semantic versioning.

**Narrative**
As anyone who interacts with Lookit (but developers in particular), I want to be able to reason about the evolution of the platform using the time-tested method of [Semantic Versioning](https://semver.org/).

**Acceptance Criteria**

- [x] Delete current Lookit-api tags
- [x] Documented process in `CONTRIBUTING.md` (see #556) for creating new release branches and pushing out new releases
- [x] Requisite invoke tasks for cutting new releases
- [x] Cut a brand new v1.0.0

**Implementation Notes**
Again, version bumps should be orchestrated through invoke tasks. This will also finally allow us to start using the last part of `git flow` that we don't actually use yet, which is the `release` branch workflow.
